f8a_worker/dispatcher/nodes.yml

```diff
@@ -455,6 +455,15 @@
         encryption: false
         versioned: true
 
+    - name: 'S3CrowdSourceTags'
+      import: 'f8a_worker.storages'
+      configuration:
+        <<: *configurationS3
+        bucket_name: '{DEPLOYMENT_PREFIX}-cleaned_tags'
```

Comment:
is this bucket name verified with Sarah?

Reply:
Yes @miteshvp. Sarah was aksing for buckt ```dev-stack-analysis-clean-data+<ecosystem>/github/data_input_raw_package_list/package_topic.json```, just a small change is here. Instead of ```dev```, ```username``` will appear here. If you will say put that hard-coded value, i will update that.

Reply:
Let the bucket name be hard-coded `dev-stack-analysis-clean-data+<ecosystem>/github/data_input_raw_package_list`. But create a new document with `crowd_sourcing_package_topic.json`